x-pack/plugin/esql/qa/testFixtures/src/main/resources/ip.csv-spec

```diff
@@ -485,3 +485,52 @@ beta         | 127.0.0.1
 beta         | 127.0.0.1
 beta         | 127.0.0.1
 ;
+
+ipPrefix
+//tag::ipPrefix[]
+required_capability: fn_ip_prefix
```

Comment:
Could you remove the `required_capability` form the captured example? Just shift the `//tag` marker below this line.